test/ExpressiveInstallerTest/SetupDataAndCacheDirTest.php

```diff
@@ -17,17 +17,17 @@ class SetupDataAndCacheDirTest extends OptionalPackagesTestCase
     /**
      * @var OptionalPackages
      */
-    protected $installer;
+    private $installer;
```

Comment:
It doesn't matter. I've changed it just for consistence. Everywhere in expressive tests we have `private` properties in test classes.